tests/boot/boot_to_desktop.pm

```diff
@@ -27,8 +27,9 @@ sub run {
     # the timeout here must cover it. UEFI DVD adds some 60 seconds on top.
     my $timeout = get_var('UEFI') ? 140 : 80;
     # Add additional 60 seconds if the test suite is migration as reboot from
-    # pre-migration system may take an additional time.
-    $timeout += 60 if get_var('PATCH') || get_var('ONLINE_MIGRATION');
+    # pre-migration system may take an additional time. Booting of encrypted disk
+    # needs additional time too.
+    $timeout += 60 if get_var('PATCH') || get_var('ONLINE_MIGRATION') || get_var('ENCRYPT');
```

Comment:
this change, independant of the other two, looks easy and good. If you keep this one separate we can merge it easily.

Reply:
Removed as well and will handle it separately.